### ISSUE TITLE
Error in the effect inference

### ISSUE THREAD

**Issue by RicardoGCCR** [tseemann/snippy#313]
Hi Tseemann, 
I am currently running snippy (v4.0.2) for an evolutionary study for our lab bacterial strain (Illumina 150bp -PE). I am using a well-annotated genome as the reference. The program seems to identify correctly the single mutation present in the evolved clones (at the NT_POS - according to the SANGER confirmation), but the effect reported is incorrect (both AA_POS and EFFECT). For instance, I am getting "stop_gained" effect in the middle of an essential gene, and when I analyzed the gene separately, the mutation does not cause such effect but a missense AA substitution. 
I went to the reference genome file (.gb) to check any issues in the gene translation, but there were no issues at all. 

So in summary: the program reports the SNP NT_pos correctly but miss-interpreting the effect. 

Any idea what can be causing this issue?

Thanks, 
Ricardo

**Comment by tseemann** [comment]
Yes, this is a previously recorded issue with `snpEff` where older versions didn't use the `frame`/`phase` of the GFF file correctly.

Many bugs have also been fixed in Snippy since the version you are using.

Please upgrade `snpEff`, and `snippy` to at least 4.4.3.  I encourage you to use Bioconda or Brew to ensure you have correctly working installations.

**Comment by RicardoGCCR** [comment]
Excellent, 
Thanks a lot for the fast response.
Best
